doc/designs/quic-design/synthesis.md

```diff
@@ -0,0 +1,489 @@
+QUIC Proposed Final MVP API Design
+==================================
+
+This incorporates the result of the DDD design exploration process and
+synthesizes the output of that process with the original QUIC API proposal. The
+the below API has been designed to support the DDD demos with their diffs
+without further changes, and thereby meet the agreed requirements for QUIC.
+
+Response to OTC Questions
+-------------------------
+
+1. *Timer based events*
+
+   This was the subject of extensive discussion in the documents accompanying
```

Comment:
Could you add a link to these?  I'm not seeing anything relevant in #17991.  Or even add the discussions to #17991 in addition to the code samples.

Reply:
I think those demos should end up as part of our documentation, so that this documentation can properly refer to them.